Console/Command/NostoClearQueueCommand.php

```diff
@@ -0,0 +1,201 @@
+    }
+
+    /**
+     * Clear DB.
+     *
+     * @param string $topicName
+     * @param SymfonyStyle $io
+     * @return void
+     */
+    private function clearDBQueues(string $topicName, SymfonyStyle $io): void
+    {
+        // Get connection.
+        $connection = $this->resourceConnection->getConnection();
+
+        // Start DB transaction.
+        $connection->beginTransaction();
+        try {
+            // Emptying DB tables.
+            $this->clearQueueMessages($topicName, $connection);
+            $this->clearRelatedRecords($topicName, $connection);
+            $connection->commit();
+        } catch (Exception $exception) {
+            $connection->rollBack();
+            $io->error('An error occurred while clearing DB queues for topic '
+                . $topicName . ': '
+                . $exception->getMessage()
+            );
+        }
+    }
+
+    /**
+     * Emptying queue message tables.
+     *
+     * @param string $topicName
+     * @param $connection
+     * @return void
+     */
+    private function clearQueueMessages(string $topicName, $connection): void
+    {
+        $queueMessageTable = $this->resourceConnection->getTableName('queue_message');
+        $queueMessageStatusTable = $this->resourceConnection->getTableName('queue_message_status');
+
+        // Get all IDs from "queue_message" table.
+        $select = $connection->select()
+            ->from($queueMessageTable, ['id'])
+            ->where('topic_name = ?', $topicName);
+        $messageIds = $connection->fetchCol($select);
+
+        // Delete related records from "queue_message_status" table.
+        if (!empty($messageIds)) {
+            $connection->delete($queueMessageStatusTable, ['message_id IN (?)' => $messageIds]);
+        }
+
+        // Delete records from "queue_message" table.
+        $connection->delete($queueMessageTable, ['topic_name = ?' => $topicName]);
+    }
+
+    /**
+     * Emptying related tables.
+     *
+     * @param string $topicName
+     * @param $connection
+     * @return void
+     */
+    private function clearRelatedRecords(string $topicName, $connection): void
+    {
+        $magentoOperationTable = $this->resourceConnection->getTableName('magento_operation');
+        $magentoBulkTable = $this->resourceConnection->getTableName('magento_bulk');
+
+        // Get all IDs from "magento_operation" table.
+        $selectBulkUuids = $connection->select()
+            ->from($magentoOperationTable, ['bulk_uuid'])
+            ->where('topic_name = ?', $topicName);
+        $bulkUuids = $connection->fetchCol($selectBulkUuids);
+
+        // Delete related records from "magento_bulk" table.
+        if (!empty($bulkUuids)) {
+            $connection->delete($magentoBulkTable, ['uuid IN (?)' => $bulkUuids]);
+        }
+
+        // Delete records from "magento_operation" table.
+        $connection->delete($magentoOperationTable, ['topic_name = ?' => $topicName]);
```

Comment:
This can be better handled by querying for the bulk ID's and using this method to delete them:
https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/AsynchronousOperations/Model/BulkManagement.php#L237

Also notice that you only want to delete the ones with status = `STATUS_TYPE_OPEN`